src/NzbDrone.Core/Notifications/Pushsafer/PushsaferSettings.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Pushsafer
+{
+    public class PushsaferSettingsValidator : AbstractValidator<PushsaferSettings>
+    {
+        public PushsaferSettingsValidator()
+        {
+            RuleFor(c => c.ApiKey).NotEmpty();
+        }
+    }
+
+    public class PushsaferSettings : IProviderConfig
+    {
+        private static readonly PushsaferSettingsValidator Validator = new PushsaferSettingsValidator();
+
+        [FieldDefinition(0, Label = "API Key", HelpLink = "https://www.pushsafer.com/dashboard")]
+        public string ApiKey { get; set; }
+
+        [FieldDefinition(1, Label = "Sound", Type = FieldType.Textbox, HelpText = "Notification sound, leave blank to use the default, set 0 for silent, set 1-50 for custom", HelpLink = "https://www.pushsafer.com/en/pushapi")]
+        public string Sound { get; set; }
+
+        [FieldDefinition(2, Label = "Icon", Type = FieldType.Textbox, HelpText = "Notification icon, leave blank to use the default, set 1-176 for custom", HelpLink = "https://www.pushsafer.com/en/pushapi")]
+        public string Icon { get; set; }
```

Comment:
Reading the description, what does 0 mean?

Reply:
0 makes have no effect The used icon will still be the default one (1)